R/clean_redcap_long.R

```diff
@@ -416,14 +421,38 @@ convert_mixed_instrument <- function(db_data_long, mixed_structure_ref) {
         )
       )
 
+    repeat_together_present <- any(
```

Comment:
Since we now want to separate RT events so their instances get captured under `redcap_event_instance`, we first need to check for the right behavior for RTs (i.e. no repeating instrument but a repeat instance detected).